_sass/vendor/pygments/index.html

```diff
@@ -0,0 +1,858 @@
+<!DOCTYPE html>
```

Comment:
Can this file be removed?

Reply:
Yes; the docs could provide a link to https://pygments.org. I should do this also in #578 when it that PR has been reviewed.